### PR TITLE
chore: add GH workflows and templates

### PR DESCRIPTION
**Pull request by phantumcode**
*Issue #, if available:*

*Description of changes:*
* Adding templates for new issues and feature request
* Add GH actions to automate Slack notifications to when issues are open, closed, and commented and when a release is completed

*Check points: (check or cross out if not relevant)*

- [ ] Added new tests to cover change, if needed
- [ ] Build succeeds with all target using Swift Package Manager
- [ ] All unit tests pass
- [ ] Security oriented best practices and standards are followed (e.g. using input sanitization, principle of least privilege, etc)
- [ ] Documentation update for the change if required
- [ ] PR title conforms to conventional commit style
- [ ] If breaking change, documentation/changelog update with migration instructions

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    id: behavior
+    attributes:
+      label: Expected behavior
+      description: A clear and concise description of what you expected to happen.
+    validations:
+      required: true
+  - type: input
+    id: liveness-version
+    attributes:
+      label: Version
```
Comment (thisisabhash) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:35]: suggestion to rename the label to something like Swift Liveness version

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      placeholder: e.g. 2.12.0
+    validations:
```
Comment (thisisabhash) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:37]: do we need the CLI version / Swift version also, like the `amplify-swift` repo?

Comment (phantumcode) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:37]: no, this shouldn't be needed

File: .github/ISSUE_TEMPLATE/bug_report.yaml (hunk 1 above)
Comment (phantumcode) [reply .github/ISSUE_TEMPLATE/bug_report.yaml]: updated per suggestion

**Review by thisisabhash — APPROVED** [review]

**Pull request merged**